qprofiler2/src/org/qcmg/qprofiler2/QProfilerSummary2.java

```diff
@@ -24,7 +24,7 @@
 
 import org.w3c.dom.Element;
 
-public class QProfilerSummary2 {
+public class QProfilerSummary2 { 
```

Comment:
same in this file - not sure why space has been added after all the opening block parentheses `{` .